.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -25,15 +23,14 @@ This PR solves #12345
 - [ ] Create tests;
 -->
 
+<!-- Also, don't forget to review your code before marking it as ready to merge -->
+
 ### Pull request checklist
 
 <!-- Before submitting the PR, please address each item -->
 
 - [ ] **Tests**: This PR includes tests for covering the features or bug fixes (if applicable).
 - [ ] **Docs**: This PR updates/creates the necessary documentation.
-- [ ] **Commits descriptions**: All commits in this PR contain a title and description according to [CONTRIBUTING.md](https://github.com/profusion/quickstart-nodejs-rest/blob/master/doc/CONTRIBUTING.md#commits).
```

Comment:
This commit check would be beneficial to enforce conventional commit guidelines. However, we always do this in the CI before committing/pushing. I wonder if we could replace this with smth like:

- [ ] **CI**: Make sure your Pull Request passes all CI checks. If not, clarify the motif behind and the action plan to solve (may reference a ticket)